utility/src/main/java/org/oppia/android/util/parser/html/HtmlParser.kt

```diff
@@ -124,11 +123,17 @@ class HtmlParser private constructor(
     }
 
     val imageGetter = urlImageParserFactory?.create(
-      htmlContentTextView, gcsResourceName, entityType, entityId, imageCenterAlign
```

Comment:
I found, that besides the two factory methods this PR is targeting, there are other unnamed variables in other functions. For example this one. Should I change them all?

Reply:
It's best to just focus on this current PR. Thanks